client/src/pages/account/index.js

```diff
@@ -1,32 +1,19 @@
-import api from 'api'
+import React from 'react'
 import { Loader } from 'components/Loader'
 import { EnterEmailModal } from 'components/modals/EnterEmailModal'
 import { useSignIn } from 'hooks/useSignIn'
-import { useRouter } from 'next/router'
-import React from 'react'
+// import { useRouter } from 'next/router'
 
-export const Account = ({ noCode, orcid, accessToken, refreshToken }) => {
-  const router = useRouter()
-  const {
-    needsEmail,
-    orcidInfo,
-    setOrcidInfo,
-    setEmail,
-    setNeedsEmail,
-    setError,
-    user
-  } = useSignIn()
-
-  // If the user navigates to /account manually, redirect them to /account/basic-information
-  if (noCode && !needsEmail && user) {
-    router.replace('/account/basic-information')
-  }
+export const Account = ({ code, originUrl }) => {
+  // const router = useRouter()
+  const { needsEmail, setEmail, setNeedsEmail, setError } = useSignIn(
+    code,
+    originUrl
+  )
 
-  React.useEffect(() => {
-    if (!orcidInfo && orcid && accessToken && refreshToken) {
-      setOrcidInfo({ orcid, accessToken, refreshToken })
-    }
-  })
+  // this page should only:
+  // 1) finish sign up by asking for email
+  // 2) let the hook redirect the user to where they left off
```

Comment:
Was this a note to yourself or did you mean to leave it here?

Reply:
this is there on purpose becuase it isnt a "view" page more of a client side endpoint that wraps up the sign in process and then bounces you to where you should go - this is just to make sure other stuff is either mentioned here or put in a  more appropriate place

Reply:
going to move this out of the component towards the top so its easier to see